integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/BuildConfigurationEndpointTest.java

```diff
@@ -482,9 +483,9 @@ public void testGetBuildTypeDefaultAlignmentParameters() throws RemoteResourceEx
         BuildConfigurationClient client = new BuildConfigurationClient(RestClientConfiguration.asAnonymous());
 
         for (BuildType buildType : BuildType.values()) {
-            Optional<String> params = client.getBuildTypeDefaultAlignmentParameters(buildType.name());
-            assertThat(params).isPresent();
-            assertThat(!params.get().isEmpty());
+            AlignmentParameters params = client.getBuildTypeDefaultAlignmentParameters(buildType.name());
+            assertThat(!params.getParameters().isEmpty());
+            assertThat(!params.getBuildType().isEmpty());
```

Comment:
I think it should be like this?
```suggestion
            assertThat(params.getParameters()).isNotEmpty();
            assertThat(params.getBuildType()).isNotEmpty();
```

Reply:
I can't find anything similar to String.isNotEmpty(), hamcrest supports something like this assertThat(string, is(not(emptyString()))) but that does not seem any better to me

Reply:
You must call it on the object returned by assertThat not String.
E.g.: assertThat("test").isNotEmpty();